moving-keys-between-servers.md

```diff
@@ -0,0 +1,58 @@
+# How to move a gpg key between servers
+
+Sometimes, you need to move a gpg key from one server to another. 
+
+But why?
+
+Let's say, you prepare a site to deploy to staging. You create a gpg key on your local machine, and you export it to the staging server. For the sake of good housing keeping, you then remove the key from your local machine. Days, weeks, months elapse - and, now, you (or someone else) needs to deploy the app to production *on a different server*.
+
+Or, you need to migrate an application from its current server to a new server: this requires the gpg key of the application to be included on the server keyring. 
+
+#### 1. On your local machine, add your private SSH key (i.e., id_rsa) to the ssh-agent.
+
+```
+ssh-add ~/.ssh/id_rsa
+```
+
+#### 2. Shell into the "old" server, and in doing so, forward the authentication credentials to the server.
```

Comment:
it might be nice to indicate in a comment in each code block where you're executing command, e.g. "local", "old server", "new server"